ui/app/AppLayouts/Chat/components/ChannelIcon.qml

```diff
@@ -0,0 +1,79 @@
+import QtQuick 2.3
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+import Qt.labs.platform 1.1
+import "../../../../imports"
+import "../../../../shared"
+
+Rectangle {
+  property var topMargin: Theme.smallPadding
+  property var bottomMargin: Theme.smallPadding
+  property var channelName
+  property var channelType
```

Comment:
Not sure if it make s a difference, but I think it's better to use real types. In this case: `int`, `int`, `string` and `int`

Reply:
Very good idea